charts/argo-cd/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: "1.2.4"
+appVersion: "1.2.5"
```

Comment:
Please leave this at `1.2.4` as it reflects the image tag used in this chart version.

Reply:
Yep my bad. Left the `appVersion` where it was after rebasing.